weekly/2025/week-13.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Week 13 in Packit
+date: 2025-03-31
+authors: nforro
+tags:
+  - 2025-March
+  - 2025
+  - March
+---
+
+## Week 13 (March 25th – March 31st)
+
+- We have fixed an issue that could cause subsequent runs
+  of `propose_downstream`/`pull_from_upstream` to fail if upstream git repo
+  contains submodules that are manipulated with in actions.
```

Comment:
```suggestion
  contains submodules that are manipulated within actions.
```

Reply:
I don't think that's correct. Let me phrase it a bit differently so you see what I mean:
`contains submodules you manipulate with during execution, for example in actions`

Reply:
ah ok, read too quickly, lgtm!